### ISSUE TITLE
Profiling with 11_3_0, 12_0_0_pre2, 12_0_0_pre3

### ISSUE THREAD

**Issue by jpata** [jpata/cms-reco-profiling-web#14]
The 12_0_0_pre2 results are now available. It would be great to propagate them to the [web page](https://cms-reco-profiling.web.cern.ch/cms-reco-profiling/)!

```bash
$ ls /eos/cms/store/user/cmsbuild/profiling/data/CMSSW_12_0_0_pre2/slc7_amd64_gcc900/
11834.21  23434.21

$ ls /eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_0_0_pre2/*
/eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_0_0_pre2/11834.21:
step3  step4

/eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_0_0_pre2/23434.21:
step3  step4
```

@xoqhdgh1002 could you please take a look and open a PR? Thanks!

**Comment by jpata** [comment]
I also added generated sql files for 11_3_0 now.

**Comment by jpata** [comment]
I also added 12_0_0_pre3 now, both the raw igprof and sql files in the usual locations.

**Comment by jpata** [comment]
kind ping @xoqhdgh1002, it would be great to update the web page with the latest profiles for 11_3_0, 12_0_0_pre2, 12_0_0_pre3.

**Comment by jpata** [comment]
There are now implemented in #18.
